### DISCUSSION TITLE
kepler map not showing

### DISCUSSION THREAD

**Discussion by Niko-La** [opengeos/leafmap#107] [Q&A] [answered]
After running
```
!pip install leafmap
!pip install keplergl

leafmap.update_package()
```

Yet kepler widget is showing the following output

![image](https://user-images.githubusercontent.com/14004559/129657687-92f770b5-3a09-41f7-b56a-029d67c7df62.png)

**Comment by giswqs** [comment, marked as answer]
Are you using Jupyter notebook or JupyterLab? This seems to be a common issue. See 

- https://github.com/giswqs/geemap/issues/67
- https://github.com/matplotlib/ipympl/issues/112
- https://github.com/jupyter-widgets/ipyleaflet/issues/739
- https://github.com/jupyter-widgets/ipyleaflet/issues/558
- https://github.com/jupyter-widgets/ipyleaflet/issues/547

Try creating a fresh conda env to install leafmap and keplergl. 

```
    conda create -n geo python=3.8
    conda activate geo
    conda install geopandas
    conda install mamba -c conda-forge
    mamba install leafmap xarray_leaflet -c conda-forge
    pip install keplergl
```
